yarn.lock

```diff
@@ -3142,7 +3142,7 @@
     lodash "^4.17.15"
     moment "^2.24.0"
 
-"@joystream/warthog@2.41.2", "@joystream/warthog@~2.41.2":
+"@joystream/warthog@2.41.2", "@joystream/warthog@^2.40.0":
```

Comment:
this change doesn't seem correct. When I ran yarn build:packages in your branch it reverted to:
`"@joystream/warthog@2.41.2", "@joystream/warthog@~2.41.2":`

Reply:
perhaps you are not using the same version of yarn. (which may be the case if you haven't installed volta)

Reply:
```
$ yarn --version
1.22.15

$ volta --version
1.0.5
```